assets/scaffold/files/additions-default.settings.txt

```diff
@@ -1,3 +1,29 @@
+// Possible values: local, dev, test, live.
+// Configuration further in this file sets different settings for different
+// environments.
+if (defined('PANTHEON_ENVIRONMENT')) {
```

Comment:
@randallquesadaa Lets remove Pantheon specific things in this PR. There should only be a "local" and "Remote" environment.